Sources/ComposableArchitecture/Store.swift

```diff
@@ -7,11 +7,19 @@ import ReactiveSwift
 /// You will typically construct a single one of these at the root of your application, and then use
 /// the `scope` method to derive more focused stores that can be passed to subviews.
 public final class Store<State, Action> {
-  @MutableProperty private(set) var state: State
+  @MutableProperty
+  private(set) var state: State
+  
   private var isSending = false
   private let reducer: (inout State, Action) -> Effect<Action, Never>
   private var synchronousActionsToSend: [Action] = []
   private var bufferedActions: [Action] = []
+  
+  @DisposablesScopedStorage
+  private var effectDisposables: [UUID: Disposable] = [:]
```

Comment:
It can be `internal` as it is in TCA so it's accessible from tests, which avoids needing `effectDisposablesCount` below just for tests.

Reply:
Fine, il'll fix it soon (probably today or tomorrow)

Reply:
Fixed